tests/test_filters.py

```diff
@@ -118,6 +120,8 @@ def create_stmt_without_col_attributes(self):
                 `id` int(11),
                 `age` int(11),
                 `name` varchar(64),
+                `size` enum('small', 'medium', 'large'),
```

Comment:
sqlparse tokens and not case normalized. against an upper case type the corresponding expected type also has to be upper case. So i believe the test wont be meaningful.